### DISCUSSION TITLE
[FEATURE] Email-Builder based on Yoopta-Editor

### DISCUSSION THREAD

**Discussion by liesislukas** [yoopta-editor/Yoopta-Editor#331] [Ideas]
### Is this a unique feature?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Is your feature request related to a problem/unavailable functionality? Please describe.

yes, export HTML for email content, so would include all needed css and would properly work on email clients

### Proposed Solution

just some helper function which would let to get email html out of the editor

### Do you want to work on this issue?

No

**Comment by Darginec05** [comment]
Hi @liesislukas 
Yes, yes and yes!

We have plans to add `deserialize/serialize` methods for email exports in `@yoopta/exports`.
And one more good news: we want to create separate package for this => Email builder with the Yoopta editor

**Comment by codythatsme** [comment]
Hey @Darginec05!

Has work on the Email Builder package started yet? I'd love to help contribute.

**Comment by Darginec05** [comment]
@CodyCodes95 that would be great!
But first, maybe let's discuss the approach to email builder?

**Comment by Darginec05** [comment]
We started working on email-builder here https://github.com/Darginec05/Yoopta-Editor/pull/339
@CodyCodes95 Do you want to join?
